packages/elements/src/slider/elements/slider.ts

```diff
@@ -1263,7 +1272,8 @@ export class Slider extends FormFieldElement {
    * @returns {void}
    */
   private onFromValueChange(): void {
-    if (this.isValueInBoundary(this.fromNumber, SliderDataName.from)) {
+    // recalculate value and check boundary before rounding
```

Comment:
Let's replace the current "what" comment why "comment" as written below.
```javascript
/** 
 * This method handles "from" value change triggered by 
 * both user interaction and programmatic update.
 * 
 * In case of user interaction update, 
 * the value would be adjusted by min range before entering this method. 
 * That adjustment needs to be reverted.
 * 
 * In case of programmatic value update, 
 * This method could handle the revert although there is no prior adjustment.
*/